python/cairo-ec/src/cairo_ec/curve/secp256k1.cairo

```diff
@@ -233,88 +243,132 @@ func try_recover_public_key{
+    // RLCSumDlogDiv for 2 points: n_coeffs = 18 + 4 * 2 = 26 (0-25)
 
     // Generator point, same as in get_generator_point()
-    assert ecip_input[30] = UInt384(
+    assert ecip_input[26] = UInt384(
         0x2dce28d959f2815b16f81798, 0x55a06295ce870b07029bfcdb, 0x79be667ef9dcbbac, 0x0
     );
-    assert ecip_input[31] = UInt384(
+    assert ecip_input[27] = UInt384(
         0xa68554199c47d08ffb10d4b8, 0x5da4fbfc0e1108a8fd17b448, 0x483ada7726a3c465, 0x0
     );
 
     // R point
-    assert ecip_input[32] = r_point.x;
-    assert ecip_input[33] = r_point.y;
+    assert ecip_input[28] = r_point.x;
+    assert ecip_input[29] = r_point.y;
 
-    assert ecip_input[34] = ep1_low_384;
-    assert ecip_input[35] = en1_low_384;
-    assert ecip_input[36] = sp1_low_384;
-    assert ecip_input[37] = sn1_low_384;
+    assert ecip_input[30] = ep1_low_384;
+    assert ecip_input[31] = en1_low_384;
+    assert ecip_input[32] = sp1_low_384;
+    assert ecip_input[33] = sn1_low_384;
 
-    assert ecip_input[38] = ep2_low_384;
-    assert ecip_input[39] = en2_low_384;
-    assert ecip_input[40] = sp2_low_384;
-    assert ecip_input[41] = sn2_low_384;
+    assert ecip_input[34] = ep2_low_384;
+    assert ecip_input[35] = en2_low_384;
+    assert ecip_input[36] = sp2_low_384;
+    assert ecip_input[37] = sn2_low_384;
 
-    assert ecip_input[42] = ep1_high_384;
-    assert ecip_input[43] = en1_high_384;
-    assert ecip_input[44] = sp1_high_384;
-    assert ecip_input[45] = sn1_high_384;
+    assert ecip_input[38] = ep1_high_384;
+    assert ecip_input[39] = en1_high_384;
+    assert ecip_input[40] = sp1_high_384;
+    assert ecip_input[41] = sn1_high_384;
 
-    assert ecip_input[46] = ep2_high_384;
-    assert ecip_input[47] = en2_high_384;
-    assert ecip_input[48] = sp2_high_384;
-    assert ecip_input[49] = sn2_high_384;
+    assert ecip_input[42] = ep2_high_384;
+    assert ecip_input[43] = en2_high_384;
+    assert ecip_input[44] = sp2_high_384;
+    assert ecip_input[45] = sn2_high_384;
 
-    // Q_low / Q_high / Q_high_shifted (filled by prover) (50 - 55).
+    // Q_low / Q_high / Q_high_shifted (filled by prover) (46 - 51).
 
-    assert ecip_input[56] = random_point.x;
-    assert ecip_input[57] = random_point.y;
+    assert ecip_input[52] = random_point.x;
+    assert ecip_input[53] = random_point.y;
 
     // a_Weierstrass
-    assert ecip_input[58] = a;
+    assert ecip_input[54] = a;
+    // b_Weierstrass
+    assert ecip_input[55] = b;
     // base_rlc
-    assert ecip_input[59] = rlc_coeff_u384;
-
-    let (add_offsets_ptr, mul_offsets_ptr) = get_full_ecip_2P_circuit();
-    assert add_mod_ptr[0] = ModBuiltin(
-        p=p, values_ptr=cast(range_check96_ptr, UInt384*), offsets_ptr=add_offsets_ptr, n=117
-    );
-    assert mul_mod_ptr[0] = ModBuiltin(
-        p=p, values_ptr=cast(range_check96_ptr, UInt384*), offsets_ptr=mul_offsets_ptr, n=108
+    assert ecip_input[56] = rlc_coeff_u384;
```

Comment:
it seems to me that you're asserting twice on the range_check96 doing this, because the assert is also done in the function itsefl